### PR TITLE
Docker fix branch

### PR DESCRIPTION
**Pull request by qreqit**

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 databaseChangeLog:
   - include:
       file: /db/changelog/changes/01-create-table-books.yaml
-  - include:
-      file: db/changelog/changes/02-create-table-shoppingCart.yaml
+#  - include:
+#      file: db/changelog/changes/02-create-table-shoppingCart.yaml
```
Comment (okuzan) [inline src/main/resources/db/changelog/db.changelog-master.yaml:5]: ? shopping card should be created

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+version: "3.8"
+services:
+  mysqldb:
+    platform: linux/amd64
+    image: mysql
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234567
+      - MYSQL_DATABASE=book-store
```
Comment (okuzan) [inline docker-compose.yml:10]: use env values

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      "spring.datasource.url" : "jdbc:mysql://mysqldb:3306/book-store",
+      "spring.datasource.username" : "root",
+      "spring.datasource.password" : "1234567",
+      "spring.jpa.properties.hibernate.dialect" : "org.hibernate.dialect.MySQLDialect",
+      "spring.datasource.driver-class-name" : "com.mysql.cj.jdbc.Driver",
+      "spring.jpa.hibernate.ddl-auto" : "update",
+      "jwt.expiration": "1800000",
+      "jwt.secret": "haveAGoodDayIfYouReadIt243509349584398534025"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (okuzan) [inline docker-compose.yml:39]: use env values everywhere

File: .env [hunk 4]
```diff
@@ -0,0 +1,13 @@
+MYSQLDB_USER=root
```
Comment (okuzan) [inline .env:1]: real .env should never in source control, add it to .gitignore, use git rm --cached .env, add .env.template with actual values removed

**Review by okuzan — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 1 above)
Comment (qreqit) [reply src/main/resources/db/changelog/db.changelog-master.yaml:5]: i created it in another file, but forgot change name

File: docker-compose.yml (hunk 2 above)
Comment (qreqit) [reply docker-compose.yml:10]: ok

File: .env (hunk 4 above)
Comment (qreqit) [reply .env:1]: i will remember it

File: .env [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+SPRING_JPA_PROPERTIES_HIBERNATE_DIALECT=org.hibernate.dialect.MySQLDialect
+SPRING_DATASOURSE_DRIVER_CLASS_NAME=com.mysql.cj.jdbc.Driver
+SPRING_JPA_HIBERNATE_DDL_AUTO=update
+
+JWT_SECRET=haveAGoodDayIfYouReadIt243509349584398534025
+JWT_EXPIRATION_MS=1800000
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+SPRING_DEBUGING_PORT=5005
```
Comment (liliia-ponomarenko) [inline .env:17]:
```suggestion
SPRING_DEBUGING_PORT=5005

```

File: src/main/java/mate/academy/springbootwebgreqit/model/User.java [hunk 6]
```diff
@@ -50,7 +50,7 @@ public class User implements UserDetails {
     @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
     private ShoppingCart shoppingCart;
 
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
```
Comment (liliia-ponomarenko) [inline src/main/java/mate/academy/springbootwebgreqit/model/User.java:53]: don't use the Eager fetch type. Better fix repository method ;)

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! A few minor comments ;)

**Pull request merged**